TEveEventDisplay/src/TEveMu2eMCInterface.cc

```diff
@@ -81,63 +94,155 @@ namespace mu2e{
     return std::count(v.begin(), v.end(), x);
   }
   
-  /*------------
+
+    /*------------Function to make label :-------------*/
+    TEveText *TEveMu2eMCInterface::GetLabel(int PDGCode, TEveMu2eCustomHelix *line, TEveMu2eCustomHelix *line_twoDXY, TEveMu2eCustomHelix *line_twoDXZ){
+      const char* pid = "pid";
+      auto t = new TEveText(pid);
+      t->SetFontSize(15);
+      if(PDGCode == +11){ 
```

Comment:
in future consider using a switch statement for this kind of logic.

Reply:
Yes that's a good idea. The code is evolving all the time, I can look into changing things in our next round of improvements

Reply:
As with my earlier comment, this is a request  for a future upgrade, not for this PR. You have a lot of repeated code in each block - put it into a free function.  That tells future maintainers that you are doing the same thing everywhere.

You can also get strings for particle names from HepPDT.  I can provide an example if you need it.